apps/web/components/SearchInput/SearchInput.tsx

```diff
@@ -117,6 +117,7 @@ const useSearch = (
               types: [
                 SearchableContentTypes['WebArticle'],
                 SearchableContentTypes['WebSubArticle'],
+                SearchableContentTypes['WebProjectPage'],
```

Comment:
is it 100% clear that all previous webprojectpages want to be searchable ?

Reply:
Yes there are only two project pages in Contentful, the Ukraine page and the "Are you traveling to Iceland" page. We want both of them to be searchable.